bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,137 @@
+				<description>Interval to query an update from the device.</description>
+				<default>60</default>
+				<unitLabel>seconds</unitLabel>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>Sets the operating mode of the Air Conditioner - one of auto/cool/dry/fan/heat/eco/turbo or 0/1/2/3/4 or on/off</description>
+		<state>
+			<options>
+				<option value="auto">Auto</option>
+				<option value="cool">Cool</option>
+				<option value="eco">Eco</option>
+				<option value="dry">Dry</option>
+				<option value="fan">Fan</option>
+				<option value="turbo">Turbo</option>
+				<option value="heat">Heat</option>
+				<option value="ont">ON</option>
+				<option value="off">OFF</option>
+				<option value="0">Auto</option>
+				<option value="1">Cool</option>
+				<option value="2">Dry</option>
+				<option value="3">Fan</option>
+				<option value="4">Heat</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="turbo">
+		<item-type>Switch</item-type>
+		<label>Turbo</label>
+		<description>Set on/off the Air Conditioner's Turbo mode.</description>
+	</channel-type>
+	<channel-type id="air">
+		<item-type>Switch</item-type>
+		<label>Air</label>
+		<description>Set on/off the Air Conditioner's Air function if applicable to the Air Conditioner model.</description>
+	</channel-type>
+	<channel-type id="dry">
+		<item-type>Switch</item-type>
+		<label>Dry</label>
+		<description>Set on/off the Air Conditioner's Dry function if applicable to the Air Conditioner model .</description>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<description>Sets the desired room temperature</description>
+	</channel-type>
+	<channel-type id="windspeed">
+		<item-type>Number</item-type>
+		<label>Wind Speed</label>
+		<description>Sets the fan speed on the Air conditioner: Auto:0, Low:1, MidLow:2, Mid:3, MidHigh:4, High:5. The number of speeds depends on the Air Conditioner model.</description>
+		<state>
+			<options>
+				<option value="0">Auto</option>
+				<option value="1">Low</option>
+				<option value="2">MidLow</option>
+				<option value="3">Mid</option>
+				<option value="4">MidHigh</option>
+				<option value="5">High</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="health">
+		<item-type>Switch</item-type>
+		<label>Health</label>
+		<description>Set on/off the Air Conditioner's Health function if applicable to the Air Conditioner model.</description>
+	</channel-type>
+
+	<channel-type id="swingvertical">
+		<item-type>Number</item-type>
+		<label>Swing Vertical</label>
+		<description>Sets the vertical swing action on the Air Conditioner: Full Swing: 1, Up: 2, MidUp: 3, Mid: 4, Mid Down: 5, Down : 6</description>
+		<state>
+			<options>
+				<option value="Full">1</option>
```

Comment:
I think you interchanged numbers and texts here.

Reply:
yep, fixed